.circleci/requirements.txt

```
@@ -2,7 +2,7 @@ Cython==0.29.5
 PyCifRW==4.4
 coverage==4.5.4
 ddt==1.2.0
-gsd==1.7.0
+gsd==1.8.1
```

Version 1.9 came out 2 hours ago, if you're wanting to use the latest version. https://pypi.org/project/gsd/
```suggestion
gsd==1.9.0
```